src/Tribe/Commerce/PayPal/Oversell/Admin_Notice_Decorator.php

```diff
@@ -50,62 +50,112 @@ public function allows_overselling() {
 	public function modify_quantity( $qty, $inventory ) {
 		$modified = $this->policy->modify_quantity( $qty, $inventory );
 
-		$output = sprintf(
-			'<p>%s</p>',
-			sprintf(
-				esc_html__(
-					'PayPal Order %1$s caused a possible oversell of tickets: %2$d available, %3$d requested; oversell policy is %4$s.',
-					'event-tickets'
-				),
-				$this->get_order_id(),
-				$inventory,
-				$qty,
-				strtolower( $this->policy->get_name() )
-			)
-		);
+		$output = $this->style();
+		$output .= $this->header_html( $qty, $inventory );
```

Comment:
We could make this class extend `Tribe__Template`, so we could move most of the HTML we have here into a `src/admin-view/` file.

Take a look at: https://github.com/moderntribe/event-tickets/blob/bucket/tribe-commerce/src/Tribe/Admin/Views.php

To see how to setup.

Reply:
I've taken that approach into account but let go for two reasons:
1. the template would be just a mangled sequence of `if` that would defeat the purpose of a "logic-less" template
2. since this notice could/should show up anywhere in the admin area (whether a ticket-able post is being edited or not) the only place to put this notice very specialized CSS style would be either a separated CSS file loaded in all the admin screens or (pretty much the same) the common (as "in the `common` package) admin file... but that seems to lead on the same path of putting everything into a `Main` file (nope, if possible); adding inline CSS on top of a ton of logic seems to defeat the purpose of templates again.

Open to suggestions about this, though.